Docker.DotNet/Microsoft.Net.Http.Client/ManagedHandler.cs

```diff
@@ -20,37 +20,28 @@ public class ManagedHandler : HttpMessageHandler
 
         public ManagedHandler()
         {
-            _socketOpener = TCPSocketOpener;
+            _socketOpener = TCPSocketOpenerAsync;
         }
 
         public ManagedHandler(StreamOpener opener)
         {
-            if (opener == null)
-            {
-                throw new ArgumentNullException(nameof(opener));
-            }
-
-            _streamOpener = opener;
+            _streamOpener = opener ?? throw new ArgumentNullException(nameof(opener));
         }
 
         public ManagedHandler(SocketOpener opener)
         {
-            if (opener == null)
-            {
-                throw new ArgumentNullException(nameof(opener));
-            }
-
-            _socketOpener = opener;
+            _socketOpener = opener ?? throw new ArgumentNullException(nameof(opener));
         }
 
         public IWebProxy Proxy
         {
             get
             {
-                if (_proxy == null)
-                {
-                    _proxy = WebRequest.DefaultWebProxy;
-                }
+                //Need check what is the netstandard replacement for this
```

Comment:
Yeah, there isn't an option yet:

![image](https://cloud.githubusercontent.com/assets/4714040/21788546/7c82b4e0-d6b6-11e6-80c8-7c9931784b01.png)

I'm not sure that it will be available in `netstandard2.0`